### PR TITLE
Behat tests for migration.

### PR DESCRIPTION
**Pull request by solker**
- Test node contents.
- Test message contents.
- Test user contents.

### CODE REVIEW

**Comment by solker** [comment]
@amitaibu behat tests finished. Ready for code review.
selection_079

**Comment by amitaibu** [comment]
Tests are failing.

File: behat/features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -67,4 +67,11 @@ public function iVisitNodePageOfType($title, $type) {
   public function iShouldHaveAccessToThePage() {
     $this->assertSession()->statusCodeEquals('200');
   }
+
+  /**
+   * @When /^I visit the path "([^"]*)"
+   */
+  public function iVisitThePath($path){
```
Comment (solker) [inline behat/features/bootstrap/FeatureContext.php]: remove

File: behat/features/repository.feature [hunk 2]
```diff
@@ -3,7 +3,7 @@ Feature: Repository
   As a privileged user
   I need to be able to view a repository page
 
-  @api
+  @api @test
```
Comment (amitaibu) [inline behat/features/repository.feature]: remove `@test`

**Comment by solker** [comment]
@amitaibu I always got this error in Travis

> Run the build script (scripts/build)
> Beginning to build drupal-org.make.                                  [ok]
> Could not download project status information from                   [error]
> http://updates.drupal.org/release-history/admin_menu/7.x

**Comment by amitaibu** [comment]
Ok, lets keep this on hold for now, and we'll try to re-test later. You can continue with other tasks

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request closed without merging**